test/univariate/continuous/normal.jl

```diff
@@ -203,3 +203,29 @@ end
 # affine transformations
 test_affine_transformations(Normal, randn(), randn()^2)
 test_affine_transformations(NormalCanon, randn()^2, randn()^2)
+
+@testset "Normal suffstats and OffsetArrays" begin
+    a = collect(-5:5)
+    wa = collect(1.0:11.0)
+
+    resulta = suffstats(Normal, a)
+    @test resulta.s ≈ 0.0
+    @test resulta.m ≈ 0.0
+    @test resulta.s2 ≈ 110.0
+    @test resulta.tw ≈ 11.0
```

Comment:
Again, maybe sufficient here to check type inference?
```suggestion
    resulta = @inferred(suffstats(Normal, a))
```